tasks/vm_state_present.yml

```diff
@@ -66,6 +67,7 @@
     mac_address: "{{ item.1.mac_address | default(omit) }}"
     profile: "{{ item.1.profile | default(omit) }}"
     network: "{{ item.1.network | default(omit) }}"
+  # When there is no profile NICs it will use vms disks
```

Comment:
same applies here vms nics would be ignored if profile nics doesn't exist. This might not affect anybody, becase i can't image situation where i wouldn't define profile nic, however comment is not correct.